modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -26,7 +26,8 @@ Follow these steps to use OpenSSL:
 Dependencies can be downloaded from https://netty.io/wiki/forked-tomcat-native.html.
 Which dependencies you need depends upon the Neo4j version.
 Each version of Neo4j ships with a version of Netty and Netty requires specific tcnative versions.
-See table below.
+Make sure to install the version that matches your OS processor.
+For more details, see the table below.
```

Comment:
It looks good! I would change `below` to an actual link to the table and add an anchor and a caption to the table.